tests/Client/CosmosCommandTests.cs

```diff
@@ -98,4 +100,103 @@ public async Task Should_list_cosmos_accounts()
         Assert.Equal(JsonValueKind.Array, accountsArray.ValueKind);
         Assert.NotEmpty(accountsArray.EnumerateArray());
     }
+
+    [Fact]
+    [Trait("Category", "Live")]
+    public async Task Should_show_single_item_from_cosmos_account()
+    {
+        var dbResult = await CallToolAsync(
+                    "azmcp-cosmos-database-list",
+                    new()
+                    {
+                { "subscription", Settings.SubscriptionId },
+                { "account-name", Settings.ResourceBaseName }
+                    });
+        var databases = dbResult.AssertProperty("databases");
+        Assert.Equal(JsonValueKind.Array, databases.ValueKind);
+        var dbEnum = databases.EnumerateArray();
+        Assert.True(dbEnum.Any());
+
+        // The agent will choose one, for this test we're going to take the first one
+        var firstDatabase = dbEnum.First();
+        string dbName = firstDatabase.ValueKind == JsonValueKind.Object
+            ? firstDatabase.GetProperty("name").GetString()!
+            : firstDatabase.GetString()!;
```

Comment:
In which scenario would the element not be an Object?

Reply:
True, I'm changing it to throw if it's not an object because that would be an unexpected result.

Reply:
If it's impossible for things to not be an Object, we don't even need to check and throw. I assume there would be some runtime exception if you tried to access a property of something that's not an Object anyway, right?

Reply:
Yes, it will throw but the message might not be as descriptive as the one I added.